swan-cern/scripts/before-notebook.d/11_htcondor.sh

```diff
@@ -5,12 +5,6 @@
 # The HT Condor environment is configured here.
 
 # HTCondor at CERN integration
-if [[ ! -n $CERN_HTCONDOR ]]
-then
-  _log "Skipping HTCondor configuration";
-  exit 0;
-fi
-
 if [[ $CERN_HTCONDOR ]]
```

Comment:
Can we add a log message that says something like "Configuring HTCondor". Same for Spark.

Reply:
Yes I added it now.